jvm/src/main/java/com/microsoft/bond/protocol/TaggedProtocolReader.java

```diff
@@ -3,71 +3,182 @@
 import com.microsoft.bond.BondDataType;
 
 /**
- * Reads a serialized payload.
+ * Reads from serialize payload encoded using a tagged protocol.
```

Comment:
Fixed

---
In reply to: [116107724](https://github.com/tstein/bond/pull/4#discussion_r116107724) [](ancestors = 116107724)